.github/workflows/rc-release.yml

```diff
@@ -0,0 +1,233 @@
+        with:
+          token: ${{ secrets.PAT_GITHUB_SERVICE_ACCT }}
+          generate_release_notes: true
+          files: |
+            zetacored-*
+            zetaclientd-*
+
+      - name: Clean Up Alpine Workspace
+        if: always()
+        shell: alpine.sh --root {0}
+        run: |
+          set -e # fail on error
+          rm -rf *
+
+      - name: Clean Up Workspace
+        if: always()
+        shell: bash
+        run: rm -rf *
+
+  announce-release:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    needs:
+      - publish-release
+      - build-alpine
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get Version
+        run: |
+          VERSION=$(cat app/setup_handlers.go | grep "const releaseVersion" | cut -d ' ' -f4 | tr -d '"')
+          echo "BINARY_VERSION=${VERSION}" >> ${GITHUB_ENV}
+
+      - name: Determine Release Type
+        id: determine_release_type
+        run: |
+          if [[ "${{ env.BINARY_VERSION }}" =~ ^v[0-9]+\.0\.0+$ ]]; then
+            echo "RELEASE_TYPE=major" >> ${GITHUB_ENV}
+          elif [[ "${{ env.BINARY_VERSION }}" =~ ^v[0-9]+\.[0-9]+\.[1-9]+$ ]]; then
+            echo "RELEASE_TYPE=minor" >> ${GITHUB_ENV}
+          else
+            echo "RELEASE_TYPE=unknown" >> ${GITHUB_ENV}
+          fi
+
+      - name: "SEND:DISCORD:MESSAGE"
+        if: steps.determine_release_type.outputs.RELEASE_TYPE == 'major'
+        uses: gzukel/CosmosComposites/send_discord_message@main
+        with:
+          discord_token: "${{ secrets.DISCORD_TOKEN }}"
+          discord_channel_id: "${{ secrets.DISCORD_CHANNEL_ID }}"
+          discord_message: |
+            Hey <@&1122981184255840306>! A new version of the ZetaChain software has been released. 
+
+            Major Version Upgrade (e.g. v5.x.x to V6.x.x) must be completed through a governance proposal. 
+            We will submit a governance proposal in the next few days.
+            More specific information including block height will be shared as part of the governance proposal.
+
+            See the release notes for more details. https://github.com/zeta-chain/node/releases/tag/${{ env.BINARY_VERSION }}
+
+      # - name: "SEND:DISCORD:MESSAGE"
+      #   if: steps.determine_release_type.outputs.RELEASE_TYPE == 'minor'
+      #   uses: gzukel/CosmosComposites/send_discord_message@main
+      #   with:
+      #     discord_token: "${{ secrets.DISCORD_TOKEN }}"
+      #     discord_channel_id: "${{ secrets.DISCORD_CHANNEL_ID }}"
+      #     discord_message: |
+      #       Hey <@&1122981184255840306>! A new version of the zetachain node has been released.
+
+      #       Minor Version Upgrade (e.g. v5.0.x to v5.1.x) can be applied without a governance proposal.
+      #       Please review the release notes for any specific upgrade instructions or considerations.
+
+      #       See the release notes for more details. https://github.com/zeta-chain/node/releases/tag/${{ env.BINARY_VERSION }}
```

Comment:
Does it make sense of keeping it commented?

Reply:
Once we get our release process a little more refined we may want to notify discord for minor version too so we can leave it in for now